docs/user-guide.md

```diff
@@ -0,0 +1,36 @@
+## Usage
+
+If that's your first time using howdoi, run the quick help
+
+```bash
+$ howdoi howdoi
+```
+
+
+Print the help manual
+```bash
+$ howdoi # "howdoi -h" also prints help
+
+
+usage: howdoi.py [-h] [-p POS] [-a] [-l] [-c] [-n NUM_ANSWERS] [-C]
```

Comment:
Please update this with the latest version of the usage from the main README (there's a couple new commands).